django/contrib/postgres/indexes.py

```diff
@@ -220,6 +220,10 @@ def get_with_params(self):
             with_params.append('fillfactor = %d' % self.fillfactor)
         return with_params
 
+    def check_supported(self, schema_editor):
+        if self.include and not schema_editor.connection.features.supports_covering_spgist_indexes:
+            raise NotSupportedError('Covering SP-GiST indexes requires PostgreSQL 14+.')
```

Comment:
`requires` => `require` I think? (Based on the singular being _SP-GiST index_ when described by the exclusion constraints)

Reply:
Thanks. It should also be singular for the exclusion constraint case:

```diff
-                'Covering exclusion constraints using an SP-GiST index requires PostgreSQL 14+.'
+                'Covering exclusion constraints using an SP-GiST index require PostgreSQL 14+.'
```